.github/workflows/brim-release.yml

```diff
@@ -0,0 +1,20 @@
+name: Brim release
+
+on:
+  push:
+    tags:
+      - v*brim*
+
+jobs:
+  macos:
+    runs-on: macos-10.15
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./brim/release
```

Comment:
nit: Would you put the string `macos` somewhere in the release name? It calls out to `brew`, so it's mac specific.